packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -457,10 +470,11 @@ class FlutterLocalFileComparator extends FlutterGoldenFileComparator with LocalC
     final Platform platform, {
     SkiaGoldClient goldens,
     LocalFileComparator defaultComparator,
+    @visibleForTesting
```

Comment:
nit: this should probably be on the same line.

Also: Does this annotation actually work for method parameters?

Reply:
Maybe just allow to always pass this in and document what it defaults to if you don't provide it.